assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-tiles.scss

```diff
@@ -115,4 +115,13 @@
 		margin: 14px 0;
 		padding: 4px 0;
 	}
+
+	.googlesitekit-audience-widget-change-groups-cta {
```

Comment:
Seeing this and `googlesitekit-km-change-metrics-cta` have the same styles, It may be good to unify the classnames and move the style definition to `assets/sass/components/global/_googlesitekit-cta-link.scss`. Small thing but will prevent desyncs in the future.

Reply:
Updated. I placed it in `assets/sass/widgets/_widget-area.scss` instead as this is where the CTA appears, I hope that is okay. Thank you!

Reply:
Even better, thanks!